### PR TITLE
BLOG: 2021 review, part 1 - Services

### PR DESCRIPTION
**Pull request by choldgraf**
This adds a new blog post to summarize some of our major accomplishments and challenges from 2021. It is not a "list" article, but rather digs into a few high-level topics. It's a bit long, but hopefully chunked up into sections so that it's easier to follow.

I welcome feedback, ideas, or edits! I'll try to post this **next Monday** if there are no objections by then.

ref https://github.com/2i2c-org/team-compass/issues/333

### CODE REVIEW

**Comment by damianavila** [comment]
More deep feedback later, but the first thing coming to my mind after a first pass is the length 😉 

Are you planning to publish this one as a whole? Maybe you can divide it into 3 parts and publish it on a M W F schedule?
I realize the whole thing is a unity somehow (this is why I am proposing a quick iteration in the publication schedule) but I am worried about the length.

Btw, great content and a really nice way to structure the sections!

**Comment by choldgraf** [comment]
I am worried about the length as well - I'd be fine with doing 3 separate blog posts if you think that'd be a nice way to chunk it up.

**Comment by damianavila** [comment]
> if you think that'd be a nice way to chunk it up.

Yep, I think so, but let's see how others feel about it...

**Review by GeorgianaElena — APPROVED** [review]
I like it a lot! +1 on splitting it in a series of blog posts.

**Comment by choldgraf** [comment]
Thanks for the comments all - here's my plan for next steps:

- Split this into 3 blog posts, I'll try to have a draft of the first today, to post tomorrow morning, then Thurs, then next Mon.
- Each post is one of the sections above, but I'll re-order a bit in order to highlight the use-cases we've enabled first. Post order will be:
  - Services
  - Infrastructure
  - Organization
- I'll try to get a draft today so we can post tomorrow 🚀

**Comment by choldgraf** [comment]
### Update

I've updated this PR so that it splits our single post into three topic-specific posts. This also gave a bit of extra room to dive into each topic. I've added **drafts** of the other two posts, but the first post (services) is ready for review/publishing. I'd love edits or suggestions if anybody has some, and will plan to publish it tomorrow California time unless somebody asks me to hold off!

File: content/posts/drafts/2021-review-org.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Challenges we faced
+
+**Iterating in the face of uncertainty**. The biggest challenge in creating a new organization is accepting the fact that there is not a single “right” way to do things. You can investigate similar organizations from which you can draw inspiration, but there’s nobody there to tell you what is “right” or “wrong”. However, you need to start somewhere, and so we have tried to adopt a process of continual reflection and improvement rather than trying to get things perfect the first time.
+
+**Working as a fully distributed team**.  2i2c's team is split across more than 6 time zones, and there are no times of the way when we're all in standard "working hours". This adds extra complexity in team communications, coordination, and meetings.  We have had to learn how to coordinate and plan asynchronously, usually via GitHub issues, Slack, and [our Team Compass](http://team-compass.2i2c.org). We have steadily gotten better at executing as a group, and we continue to improve and iterate!
```
Comment (sgibson91) [inline content/posts/drafts/2021-review-org.md]:
```suggestion
**Working as a fully distributed team**.  2i2c's team is split across more than 6 time zones, and there are no times of the day when we're all in standard "working hours". This adds extra complexity in team communications, coordination, and meetings.  We have had to learn how to coordinate and plan asynchronously, usually via GitHub issues, Slack, and [our Team Compass](http://team-compass.2i2c.org). We have steadily gotten better at executing as a group, and we continue to improve and iterate!
```

**Review by sgibson91 — APPROVED** [review]
LGTM!

I would maybe add intro/outro paragraphs to posts 2 and 3 that say something like "these are the previous posts in this series"/"this is what's next up in the series", or " this concludes our series". But that doesn't have to be addressed until you're ready to publish them :)

**Comment by choldgraf** [comment]
Thanks for the comments all - I've made some minor updates to the post, and will merge it in now! Agreed @sgibson91 about the other two posts needing more work, I will tackle those in a follow-up PR. They won't show up in the blog for now!

**Pull request merged**